src/main/java/io/github/railroad/Railroad.java

```diff
@@ -46,8 +47,23 @@ public static Stage getWindow() {
         return window;
     }
 
+    public static void setStyle(String theme) {
+        getScene().getStylesheets().clear();
```

Comment:
This is not great as a long term solution as it clears all stylesheets even if they're not necessarily themes. Instead we should consider having some kind of container holding what the current theme is and that way we can remove that specific theme from the sheets. Possibly consider a global atomic reference maybe?